ComfyUI/tests/test_disk_quota_enforcement.py

```
@@ -0,0 +1,28 @@
+import pytest
+from unittest import mock
+import builtins
+
+def check_temp_disk_quota(min_free_gb=1):
+    # Get the free disk space in bytes for the current directory
+    free_bytes = shutil.disk_usage(".").free
+    # Convert free bytes to gigabytes
+    free_gb = free_bytes / (1024 ** 3)
+    # If free space is less than the minimum required, print a message and exit
+    if free_gb < min_free_gb:
+        print("Not enough disk space")
```

Ruff (0.11.9)

`print` found

Remove `print`

(T201)